src/model/channel/message.rs

```diff
@@ -72,6 +72,15 @@ pub struct Message {
     /// Array of [`Role`]s' Ids mentioned in the message.
     pub mention_roles: Vec<RoleId>,
     /// Channels specifically mentioned in this message.
+    ///
+    /// **Note**:
+    /// Not all channel mentions in a message will appear in mention_channels.
+    /// Only textual channels that are visible to everyone in a lurkable guild will ever be included.
+    /// Only crossposted messages (via Channel Following) currently include mention_channels at all.
```

Comment:
```suggestion
    /// Only crossposted messages (via Channel Following) currently include `mention_channels` at all.
```
As mentioned, Rust items should be `tick'd`.